CONTRIBUTING.md

```diff
@@ -56,7 +56,12 @@ Contributions are required to follow these style guides:
 │     │     └── ts: Home for all typescript-based contrib function source code
 │     └── examples: Home for all contrib function examples.
 ├── scripts
-└── tests: Home for e2e tests
+├── tests: Home for e2e tests
+└── docker
```

Comment:
I'd suggest to add them under `build/docker/`. If we have more stuff related to build in the future, we can put them in `build`.

Reply:
Done